Sources/NIO/EventLoop.swift

```diff
@@ -395,6 +395,20 @@ extension EventLoop {
     }
 }
 
+// to be removed before 2.0
+extension EventLoop {
+    @available(*, deprecated, renamed: "makeFailedFuture(_:)")
+    public func makeFailedFuture<T>(error: Error) -> EventLoopFuture<T> {
+        return self.makeFailedFuture(error)
+    }
+
+    @available(*, deprecated, renamed: "makeSucceededFuture(_:)")
+    public func makeSucceededFuture<Success>(result: Success) -> EventLoopFuture<Success> {
+        return self.makeSucceededFuture(result)
+    }
+
+}
```

Comment:
Why aren't these in the shims module?

Reply:
because the API they shim isn't NIO 1 API but API introduced in NIO 2... I filed #779 to remove them before 2.0.0